src/Modules/AuthModule/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,6 +1,251 @@
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.svg";
+import { FormDataChangPass } from "../../../../interfaces/Auth";
+import "../Login/Login.css";
+import { useContext, useState } from "react";
+import { toast } from "react-toastify";
+import axios from "axios";
+import { AuthContext } from "../../../../context/AuthContext";
 
 export default function ChangePassword() {
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    watch,
+  } = useForm<FormDataChangPass>();
+
+  const { baseUrl } = useContext(AuthContext);
```

Comment:
What is the difference between having the baseURL in a context and in a constants file?